assets/js/base/components/cart-checkout/form/index.js

```diff
@@ -9,9 +9,20 @@ import PropTypes from 'prop-types';
  */
 import './style.scss';
 
-const CheckoutForm = ( { className, children } ) => {
+const CheckoutForm = ( {
+	className,
+	children,
+	onSubmit = ( event ) => void event,
```

Comment:
Do we want to expose the event to the client code? We probably don't need to, so we could consider keeping that internal to `CheckoutForm`, and just expect a generic callback with no params.

Not a blocker - I could imagine it might be useful to expose this to client in some circumstances.

Reply:
It's common to pass along the event.